observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -165,3 +181,154 @@ spec:
+  size: 1x.extra-small
+  storage:
+    schemas:
+    - effectiveDate: '2022-06-01'
+      version: v13
+    secret:
+      name: logging-loki-s3
+      type: s3
+    storageClassName: gp3-csi
+  tenants:
+    mode: openshift-logging
+----
+
+. Create a service account for the collector:
++
+[source,shell]
+----
+$ oc create sa collector -n openshift-logging
+----
+
+. Create a `ClusterRole` for the collector:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: logging-collector-logs-writer
+rules:
+- apiGroups:
+  - loki.grafana.com
+  resourceNames:
+  - logs
+  resources:
+  - application
+  - audit
+  - infrastructure
+  verbs:
+  - create
+----
+
+. Bind the `ClusterRole` to the service account:
++
+[source,shell]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z collector
+----
+
+. Install the Cluster Observability Operator.
+
+. Create a `UIPlugin` to enable the Log section in the Observe tab:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1alpha1
+kind: UIPlugin
+metadata:
+  name: logging
+spec:
+  type: Logging
+  logging:
+    lokiStack:
+      name: logging-loki
+----
+
+. Add additional roles to the collector service account:
++
+[source,shell]
+----
+$ oc project openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector
+----
+
+. Create a `ClusterLogForwarder` CR to configure log forwarding:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: my-logforwarder
+  namespace: my-app-namespace
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled" # <1>
+spec:
+  outputs:
+    - type: lokistack # <2>
+      name: loki-otlp
+      lokiStack:
+        target:
+          name: my-lokistack
+          namespace: openshift-logging
+        dataModel: Otel # <3>
+        tuning:
+          compression: none
+        authentication:
+          token:
+            from: serviceAccount
+      tls: # <4>
+        ca:
+          key: service-ca.crt (7)
+          configMapName: openshift-service-ca.crt (7)
+  pipelines:
+   - name: my-pipeline
+     inputRefs:
+     - application
+     - infrastructure
+     outputRefs:
+     - loki-otlp
+  serviceAccount:
+    name: logger-admin
+----
+<1> Use the annotation to enable the `Otel` output, which is a Technology Preview feature.
+<2> Define the `OTLP` output type as `lokistack`.
+<3> DataModel is `Otel`.
+<4> `tls` spec identifies the keys and secret to the respective certificates that they represent.
++
+[NOTE]
+====
+You cannot use `lokistack` `labelKeys` when `dataModel` is `Otel`.
+====
+
+. Verify that logs are visible in the Log section of the Observe tab in the OpenShift web console.
```

Comment:
I believe a better way to check if OTLP was working correctly would be to check the "Distributor - Structured Metadata"  under the "OpenShift Logging / Lokistack / Writes" dashboard.

Reply:
Updated as:

* Verify that OTLP is functioning correctly by going to *Observe* > *OpenShift Logging* > *LokiStack* > *Writes* in the OpenShift web console, and checking *Distributor - Structured Metadata*.